### PR TITLE
updating labeller with mentoring v2 folder structure

### PR DESCRIPTION
**Pull request by nate-double-u**
fixes https://github.com/cncf/mentoring/issues/923

/cc @sarvjeetrajvansh @jeefy @RobertKielty for review 🙂

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -4,42 +4,42 @@
...
-  - summerofcode/2023.md
-  - /seasonofdocs/README.md # TODO, update next cycle
-  - lfx-mentorship/2023/*
-  - lfx-mentorship/2023/**/*
-  - TAGCS-Mentoring-WG/2023-meeting-minutes.md
+  - programs/summerofcode/2023.md
+  - /programs/seasonofdocs/README.md # TODO, update next cycle
+  - programs/lfx-mentorship/2023/*
+  - programs/lfx-mentorship/2023/**/*
+  - /mentoring-wg/TAGCS-Mentoring-WG/2023-meeting-minutes.md
```
Comment (aliok) [inline .github/labeler.yml:30]:
```suggestion
  - /mentoring-wg/2023-meeting-minutes.md
```

**Review by aliok — COMMENTED** [review]
Just found one little thing, otherwise LGTM

Related: https://github.com/cncf/mentoring/issues/933

File: .github/labeler.yml (hunk 1 above)
Comment (nate-double-u) [reply .github/labeler.yml:30]: good catch, thx

**Review by aliok — APPROVED** [review]
/lgtm

I guess we don't have to hold for pinged people.

**Pull request merged**